src/viam/sdk/rpc/dial.hpp

```diff
@@ -11,11 +11,76 @@
 namespace viam {
 namespace sdk {
 
-class DialOptions;
+class Credentials {
+   public:
+    /// @deprecated use `Credentials(std::string type, std::string payload)`
```

Comment:
this shouldn't be here anymore, it was removed in [this commit](https://github.com/viamrobotics/viam-cpp-sdk/pull/491)